### DISCUSSION TITLE
Querydsl generated sources will not be generated

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [querydsl/querydsl#3574] [Q&A]
I've updated the Java version of my project from Java16 to 17. In the meantime I've changed the version of the Querydsl dependency.
```
 <dependency>
            <groupId>com.querydsl</groupId>
            <artifactId>querydsl-apt</artifactId>
            <scope>provided</scope>
            <version>5.0.0</version>
            <classifier>jakarta</classifier>
        </dependency>
        <dependency>
            <groupId>com.querydsl</groupId>
            <artifactId>querydsl-jpa</artifactId>
            <version>5.0.0</version>
            <classifier>jakarta</classifier>
</dependency>
```

There are some classes that will be generated, but after the update it won't be generated, with this error message:
`java: Attempt to recreate a file for type hu.myproject.model.entity.editor.QAbstractEditor`

I've also added this plugin to the pom.xml:
 ```
 <plugin>
                <groupId>com.mysema.maven</groupId>
                <artifactId>apt-maven-plugin</artifactId>
                <version>1.1.3</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>process</goal>
                        </goals>
                        <configuration>
                            <outputDirectory>target/generated-sources/java</outputDirectory>
                            <processor>com.querydsl.apt.jpa.JPAAnnotationProcessor</processor>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

How could I make this work? Did I miss something?

**Comment by simasch** [comment]
Same problem here but only in a Maven multi-module project in a single module project it works 🤷‍♂️

**Comment by arnosthavelka** [comment]
`apt-maven-plugin` shouldn't be needed -> see https://stackoverflow.com/questions/70390931/upgrade-to-springboot-2-6-1-with-querydsl-jpa-5-0-0. IMHO your setup should be sufficient -> see my article https://dzone.com/articles/upgrade-guide-to-spring-boot-3-for-spring-data-jpa-3-and-querydsl-5

can you provide more details?

**Comment by simasch** [comment]
The classes are not generated in a multi-module Maven project. See also https://github.com/querydsl/querydsl/issues/3599

**Reply by arnosthavelka** [reply]
see my reply in #3599 😉
